docs/README.md

```diff
@@ -0,0 +1,40 @@
+## Wallet-lib
```

Comment:
Update for DAPI-client (copy/pasted from Wallet-lib)?

Reply:
Updated !